local-volume/helm/provisioner/values.yaml

```diff
@@ -0,0 +1,24 @@
+common:
+  namespace: default
```

Comment:
I would add comments to all of these parameters, describing what it's for.

Reply:
Added comment to the values.yaml. imho it makes sense more confusing and less readable.